modules/post/multi/gather/dns_srv_lookup.rb

```diff
@@ -10,7 +10,7 @@ def initialize(info={})
     super( update_info( info,
         'Name'          => 'Multi Gather DNS Service Record Lookup Scan',
         'Description'   => %q{
-          Enumerates know SRV Records for a given domaon using target host DNS query tool.
+          Enumerates know SRV Records for a given domain using target host DNS query tool.
```

Comment:
*known

Outside the scope of this original effort, but line 23 has output of 'ro' that should be 'to', methinks...